.github/workflows/prompt-test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         id: share-result
         if: always()
         run: |
-          output=$(pnpm dlx promptfoo share --yes)
+          output=$(pnpm dlx promptfoo share)
```

Comment:
👍

Reply:
I could not resolve the error and will try at another time.🙏

Reply:
Sorry! I'll work on it